playbooks/common/openshift-cluster/config.yml

```diff
@@ -31,6 +31,6 @@
 
 - include: ../openshift-master/config.yml
 
-- include: additional_config.yml
-
 - include: ../openshift-node/config.yml
+
+- include: additional_config.yml
```

Comment:
Oy, yeah as things stand some of that will need to be separated. I'd like to see service accounts become dependencies on the roles that needs them and maybe flannel be a conditional dep for master.